features/step_definitions/category_steps.rb

```diff
@@ -1,20 +1,21 @@
 Then(/^I should see the category named (.*) as the (.*) category in (.*)$/) do |category, nth, heading|
-  page.should have_xpath("//div/strong/em[text()='#{heading}']/../../following-sibling::div[#{nth[0..-3]}]/label[text()='#{category}']")
+  page.should have_xpath("//h5[contains(., '#{heading}')]/following-sibling::div[#{nth[0..-3]}]/label[text()[contains(.,'#{category}')]]")
```

Comment:
I know they were in here from the start, but I wonder if we can remove the specific reference to "h5" elements?  Just that when/if we want to change the styling in the future this will break ...